	The supplied g++-4.2 compiler on 32-bit MacOSX apparently
	mishandled private enum in the implementation (since newer
	compilers are ok with it).  This can be removed when we drop
	support for 32-bit MacOSX.
	http://tug.org/pipermail/tlbuild/2016q1/003438.html
	(and starting at http://tug.org/pipermail/tlbuild/2016q1/003433.html)

--- source/i18n/dayperiodrules.h~	2021-10-28 09:04:57.000000000 -0700
+++ source/i18n/dayperiodrules.h	2022-01-16 14:47:58.135604382 -0800
@@ -56,5 +56,5 @@
     double getMidPointForDayPeriod(DayPeriod dayPeriod, UErrorCode &errorCode) const;
 
-private:
+/* nothing private to placate old compilers --karl/koch private: */
     DayPeriodRules();
 
--- source/i18n/unicode/dtitvinf.h~	2021-10-28 09:04:57.000000000 -0700
+++ source/i18n/unicode/dtitvinf.h	2022-01-16 14:47:58.136604372 -0800
@@ -330,5 +330,5 @@
 
 
-private:
+/* nothing private to placate old compilers --karl/koch private: */
     /**
      * DateIntervalFormat will need access to
@@ -365,5 +365,6 @@
     };
 public:
-#ifndef U_HIDE_INTERNAL_API
+/* nothing private to placate old compilers --karl/koch */
+/* ifndef U_HIDE_INTERNAL_API */
     /**
      * Max index for stored interval patterns
@@ -373,6 +374,6 @@
          kMaxIntervalPatternIndex = kIPI_MAX_INDEX
      };
-#endif  /* U_HIDE_INTERNAL_API */
-private:
+/* endif */  /* U_HIDE_INTERNAL_API */
+/* nothing private to placate old compilers --karl/koch private: */
 
 
